src/main/java/org/opensearch/knn/jni/JNIService.java

```diff
@@ -42,7 +42,7 @@ public static void createIndex(int[] ids, float[][] data, String indexPath, Map<
             return;
         }
 
-        throw new IllegalArgumentException("CreateIndex not supported for provided engine");
+        throw new IllegalArgumentException("CreateIndex not supported for provided engine : " + engineName);
```

Comment:
You've changed other places, but in this line we still using direct string concatenation.

Reply:
oops changed